arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift

```diff
@@ -23,32 +23,40 @@ class GraphicsOverlayDictionaryRenderer3DViewController: UIViewController {
     /// The scene view managed by the view controller.
     @IBOutlet weak var sceneView: AGSSceneView! {
         didSet {
-            sceneView.scene = AGSScene(basemapType: .imagery)
+            sceneView.scene = AGSScene(basemapStyle: .arcGISTopographic)
```

Comment:
I used the topo base style here because it makes the symbology stand out. Also to show off the `AGSScene` vector basemap support in U13.